pkg/aspect/clean/linux.go

```diff
@@ -0,0 +1,46 @@
+//go:build linux
```

Comment:
I guess this comment tells the `go build` tool to include a different file depending on the host platform? does this function under Bazel at all?

Reply:
Correct. This tells `go build` to include this file when building the binary for `linux`.

What do you mean by "Does this function under bazel at all?". At the moment it appears to be working. `//release:aspect` which builds the binaries for darwin / linux / windows appears to be picking up the correct files as long as I can tell. And I can confirm that the darwin one is being picked up for my local build. Have not conclusively tested for windows and linux.

Reply:
We need automated tests for this, so we can request CI to do the integration testing for multiple platforms.